src/Alert/Alert.js

```diff
@@ -24,21 +24,27 @@ const Alert = ({ children, onDismiss, type }) => {
   return (
     <div className={alertClass}>
       {onDismiss &&
-        <button type='button' className='close' aria-hidden='true' onClick={onDismiss}>
-          <span className='pficon pficon-close' />
+        <button
+          type="button"
```

Comment:
i think we are safe to merge this...just wanted the clarification on it before I go disable that old plugin ;)